source/features/pagination-hotkey.tsx

```diff
@@ -21,19 +22,34 @@ const previousPageButtonSelectors = [
 function init(): void {
 	const createNextPageButton = select(nextPageButtonSelectors);
 	if (createNextPageButton) {
-		createNextPageButton.dataset.hotkey = 'ArrowRight';
+		createNextPageButton.dataset.hotkey = pageDetect.isRepoConversationList() ? 'ArrowRight' : 'n';
 	}
 
 	const createPreviousPageButton = select(previousPageButtonSelectors);
 	if (createPreviousPageButton) {
-		createPreviousPageButton.dataset.hotkey = 'ArrowLeft';
+		createPreviousPageButton.dataset.hotkey = pageDetect.isRepoConversationList() ? 'ArrowLeft' : 'p';
 	}
 }
 
 void features.add(__filebasename, {
-	shortcuts: {
-		'→': 'Go to the next page',
-		'←': 'Go to the previous page',
-	},
+	shortcuts: (pageDetect.isRepoConversationList())
+		? {
+			'←': 'Go to the previous page',
+			'→': 'Go to the next page',
+		}
+		: {
+			n: 'Go to the next page',
+			p: 'Go to the previous page',
+		},
+	include: [
+		pageDetect.isConversationList,
+		pageDetect.isLabelList,
+		pageDetect.isNotifications,
```

Comment:
there is `.paginate-container.BtnGroup > :child` that would work for Notifications as well. It seems to align visually with the other selectors, except the `a.next_page` selector.